letta/llm_api/openai.py

```diff
@@ -174,6 +174,7 @@ def openai_chat_completions_process_stream(
     stream_interface: Optional[Union[AgentChunkStreamingInterface, AgentRefreshStreamingInterface]] = None,
     create_message_id: bool = True,
     create_message_datetime: bool = True,
+    override_tool_call_id: bool = True,
```

Comment:
@sarahwooders we can turn this off if we want to use OpenAI's tool call ID tags

I'm not sure if there's any particular benefit to overriding it, other than having consistent identifiers stored in our DB?

At the same time, our code for `get_tool_call_id` doesn't even use the slug style of our other primary key generators, so idk?